tests/test_micro.py

```diff
@@ -187,6 +187,7 @@ def second(first):
     dag.build()
 
 
+@pytest.mark.xfail(reason="_capture module has been deprecated")
```

Comment:
once we delete the _micro file, we'll see some tests failing, we should delete them instead of marking them with xfail